### ISSUE TITLE
Build failure in dev afl-forkserver.c (afl undeclared)

### ISSUE THREAD

**Issue by rchildre3** [AFLplusplus/AFLplusplus#697]
at commit b850951c726258053c5635d6597704cf346fe3c4
```sh
make -j distrib
# cc ... -c src/afl-common.c
# cc ... -c src/afl-sharedmem.c
# cc -O3 -funroll-loops -march=native -D_FORTIFY_SOURCE=2 -g -Wno-pointer-sign -Wno-variadic-macros -Wall -Wextra -Wpointer-arith -I include/ -DAFL_PATH=\"/usr/local/lib/afl\" -DBIN_PATH=\"/usr/local/bin\" -DDOC_PATH=\"/usr/local/share/doc/afl\" -D_FORTIFY_SOURCE=2 -flto -c src/afl-forkserver.c -o src/afl-forkserver.o
# src/afl-forkserver.c: In function ‘afl_fsrv_start’:
# src/afl-forkserver.c:487:10: error: ‘afl’ undeclared (first use in this function)
#  487 |     if (!afl->debug || !getenv("ASAN_OPTIONS"))
#      |          ^~~
```

**Comment by rchildre3** [comment]
fixed in 040bf5a61db5fa939c6e2a884207f18b62bf1522

**Comment by domenukk** [comment]
Thanks for reporting :)
